index.cds

```diff
@@ -0,0 +1,2 @@
+namespace sap.capire.sflight; //> important for reflection
+using from './db/schema';
```

Comment:
wouldn't you also need `using`s for the files in `srv`?

Reply:
I don't think so. According to this [guide ](https://cap.cloud.sap/docs/guides/extensibility/composition#index-cds) it seems like just enough to have index.cds as the entry point

Reply:
But the example given in the [guide](https://cap.cloud.sap/docs/guides/extensibility/composition#index-cds) _does_ have `using from` also for the service files -> ?

Reply:
The idea of the `index.cds` is that it controls what _external_ users of this package should have access to via `using {...} from '@capire/sflight'`. If we'd only want them to reuse our domain model, then we could keep it like that. If we'd also allow them to reuse our services (locally or remotely), then we should include them in the `index.cds`, ...